packages/story-editor/src/app/highlights/quickActions/useQuickActions.js

```diff
@@ -180,51 +107,11 @@ export const MediaPicker = ({ render, ...props }) => {
     [selectedElements, updateElementsById]
   );
 
-  const handleMediaSelect = useCallback(
-    (resource) => {
-      try {
-        if (isTranscodingEnabled && canTranscodeResource(resource)) {
-          if (transcodableMimeTypes.includes(resource.mimeType)) {
-            optimizeVideo({ resource });
-          }
-
-          if (resource.mimeType === 'image/gif') {
-            optimizeGif({ resource });
-          }
-        }
-        // WordPress media picker event, sizes.medium.sourceUrl is the smallest image
-        insertMediaElement(
-          resource,
-          resource.sizes?.medium?.sourceUrl || resource.src
-        );
-
-        postProcessingResource(resource);
-      } catch (e) {
-        showSnackbar({
-          message: e.message,
-          dismissable: true,
-        });
-      }
-    },
-    [
-      isTranscodingEnabled,
-      canTranscodeResource,
-      insertMediaElement,
-      postProcessingResource,
-      transcodableMimeTypes,
-      optimizeVideo,
-      optimizeGif,
-      showSnackbar,
-    ]
-  );
   return (
-    <MediaUpload
+    <MediaUploadButton
```

Comment:
Neat, always nice to see code removals like this. And it works without any issues?